rasa/nlu/test.py

```diff
@@ -330,7 +330,7 @@ def plot_attribute_confidences(
         if getattr(r, target_key) != getattr(r, prediction_key)
     ]
 
-    plot_utils.plot_histogram([pos_hist, neg_hist], title, hist_filename)
+    plot_utils.plot_comparative_histogram([pos_hist, neg_hist], title, hist_filename)
```

Comment:
Please preserve the original method name (`plot_histogram`) for backwards compatibility

Reply:
Done